### ISSUE TITLE
RDP doesn't work for windows machine

### ISSUE THREAD

**Issue by Moshezauros** [Azure-Samples/compute-dotnet-manage-virtual-machine-using-vm-extensions#5]
Hello,

Thanks for these examples, they really helped my team get started with Azure.

I've ran the example (without the deleting the resource group and without removing the VmAccess extension), I then tried to RDP to the windows machine and was unable (got wrong password for all users). After resetting password for a user (using the portal) I was finally able rdp to the sample's machine.

tl; dr:
**Expected Behavior**
RDP to windows machine should work with all three created users

**Actual**
Can't login to machine

**Comment by wx-Yao** [comment]
Yeah I have the same issue. Is there any change for the underlying API call? could someone please update the sample ... if anything's broken? thanks!
